README.md

```diff
@@ -1,15 +1,22 @@
 # Save Draft
 
-`Save Draft` replaces the default "Save" button on the content creation page and 
-removes the "Published" checkbox under "Publishing options". Instead it presents 
-a "Publish" button and a "Save Draft" button for the same functionality with 
-improved usability. "Publish" will save in published form and "Save as Draft" 
-will save in unpublished form with a single click.
+`Save Draft` improves the usability of the content create/add page by removing
+the "Published" checkbox under "Publishing options" tab, and replacing the
+default "Save" button with a set of "Publish"/"Save Draft" buttons instead.
+"Publish" will save the content in published status, and "Save as Draft" will
+save it in unpublished status. The functionality remains the same, while
+requiring a single click instead of multiple ones.
 
-![Save Draft](https://github.com/backdrop-contrib/save_draft/blob/1.x-1.x/images/save_draft.gif "Save Draft example")
+`Save Draft` supports the new feature that was introduced in Backdrop core after
+version 1.10, which allows scheduled publishing of content. When a piece of
+content is scheduled to be published, the "Publish" button changes to "Schedule
+publication" button instead:
```

Comment:
Technically, when "Schedule for later" is selected, both **Publish** and **Save as Draft** are replaced by **Schedule for Publication**.